Mage.Sets/src/mage/cards/s/StrefanMaurerProgenitor.java

```diff
@@ -63,7 +64,9 @@ public StrefanMaurerProgenitor(UUID ownerId, CardSetInfo setInfo) {
         // At the beginning of your end step, create a Blood token for each player who lost life this turn.
         this.addAbility(
                 new BeginningOfEndStepTriggeredAbility(
-                        new StrefanMaurerProgenitorCreateBloodTokensEffect(),
+                        new CreateTokenEffect(
+                                new BloodToken(),
+                                StrefanMaurerProgenitorNumberPlayersLostLifeDynamicValue.instance).setText("create a Blood token for each player who lost life this turn"),
```

Comment:
I think I can get rid of this .setText. But will need more fiddling

Reply:
I've removed a few more things, but it's currently giving me this. How can I replace "instance" with "a"?

![Screen Shot 2022-02-03 at 8 20 07 PM](https://user-images.githubusercontent.com/48962821/152456680-31dd96f5-9bd9-4129-8a35-e92e11e3ec98.png)

Reply:
overwrite the toString method in your dynamic value and return "1".  You can look at setText in CreateTokenEffect to see how the string gets generated.